sdk/test-utils/recorder-new/recordings/browsers/proxy_tool/recording_sample_response.json

```diff
@@ -16,16 +16,16 @@
         "Sec-Fetch-Mode": "cors",
         "Sec-Fetch-Site": "same-site",
         "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/93.0.4577.0 Safari/537.36",
-        "x-ms-client-request-id": "f64f38b1-ff75-45be-96ec-e02716a0cef6",
```

Comment:
Unless I'm missing something, these recordings get regenerated every time we run the tests here, creating this messy diff. Should we just put the recordings folder for this project in a `.gitignore`?

Reply:
Fine with that